tests/regression/36-apron/79-mukherjee_DoubleLock_P3.c

```diff
@@ -0,0 +1,40 @@
+// SKIP
```

Comment:
Te assert fails when running with GCC.
Are you sure it was `unsigned int count` @ivanazuzic?